test/JDBC/input/schema_resolution_func-vu-prepare.sql

```diff
@@ -0,0 +1,41 @@
+create schema schema_resolution_func_s1;
```

Comment:
Please use filename_xxx for object names.

Reply:
Since the file name is `schema_resolution_func-vu-prepare` which is too long so I have prefixed `schema_resolution_func` for the object names.

Reply:
you mean it is so long that it does not work or just looks very long? I would still prefer to use long name if the work

Reply:
It works but the names look very long. I will go with the long names and update it.